scylla/Cargo.toml

```diff
@@ -25,6 +25,7 @@ thiserror = "1.0"
 itertools = "0.10.0"
 bigdecimal = "0.2.0"
 num-bigint = "0.3"
+tracing = "0.1.25"
```

Comment:
any particular reason why scylla uses tracing `0.1.25` and examples use `0.1`?

Reply:
no, I think it's a typo
fixed and updated tracing-subscriber to `0.2.16`